docs/site/components/docs/docs.js

```diff
@@ -266,7 +266,15 @@ angular
     }
 
     function compareMethods(a, b) {
-      return a.constructor ? -1: a.name > b.name ? 1 : -1;
+      if (a.constructor) {
+        return -1;
+      }
+
+      if (b.constructor) {
+        return 1;
```

Comment:
This was previously missing. If B is a constructor, it's more important than A, unrelated to its name. This previously worked because all constructors were capitalized, thus ranked higher than anything lowercase. The `gcloud` "constructor" isn't meant to be instantiated, but it is the "head" documentation point for the module.